packages/core/src/contracts/tcr/civilTCR.ts

```diff
@@ -87,9 +108,85 @@ export class CivilTCR extends BaseWrapper<CivilTCRContract> {
     return this.instance.parameterizer.callAsync();
   }
 
-  /**
-   * Event Streams
-   */
+  //#region EventStreams
+
+  //#region RawEventStreams
+
+  public listingApplications(
+    listingAddress: EthAddress,
+  ): Observable<TimestampedEvent<DecodedLogEntryEvent<ApplicationArgs, string>>> {
```

Comment:
Ops, forgot to add EventLogs

`DecodedLogEntryEvent<ApplicationArgs, string>` -> `ApplicationLogEvent`

int the next pr